manifests/init.pp

```diff
@@ -103,6 +105,7 @@
   Boolean           $manage_service,
   Boolean           $external_service,
   Boolean           $purge_config_files,
+  Boolean           $dont_linebreak_actions,
```

Comment:
The default value go here:

```suggestion
  Boolean           $dont_linebreak_actions = false,
```

The line would then be move with the other ones with a default value because by convention we list variables without default before variables with default.